lib/hexo/locals.js

```diff
@@ -48,10 +48,8 @@ class Locals {
 
   toObject() {
     const result = {};
-    const keys = Object.keys(this.getters);
 
-    for (let i = 0, len = keys.length; i < len; i++) {
-      const key = keys[i];
+    for (const key of this.getters.keys()) {
```

Comment:
Should we use `for..of` to replace traditional `for` loop? Last time I checked the `for..of` is nearly 20% slower than traditional `for` loop.

Reply:
The return value of `Map#keys()` is Iterable, not Array. This is different than `Object.keys()`.
It's interesting how fast I get if I wrap it in `Array.from()` and do it in a for loop :)